bgpd/bgp_rpki.c

```diff
@@ -1,22 +1,23 @@
-/* BGP RPKI
- * Copyright (C) 2013 Michael Mester (m.mester@fu-berlin.de)
+/*
+ * BGP RPKI
+ * Copyright (C) 2013 Michael Mester (m.mester@fu-berlin.de), for FU Berlin
+ * Copyright (C) 2014-2017 Andreas Reuter (andreas.reuter@fu-berlin.de), for FU Berlin
+ * Copyright (C) 2016-2017 Colin Sames (colin.sames@haw-hamburg.de), for HAW Hamburg
+ * Copyright (C) 2017 Marcel Röthke (marcel.roethke@haw-hamburg.de), for HAW Hamburg
  *
- * This file is part of Quagga
```

Comment:
shouldn't this file still have the line 

```
* This file is part of FRRouting.
```

as all others, too?

Reply:
Yes it should …

and now it does.